resources/spec.ejs

```diff
@@ -27,8 +27,22 @@ mkdir -p %{buildroot}/usr/
 <% }) } else { %>/usr/share/pixmaps/<%= name %>.png
 <% } %>
 
+<% if (pre) { %>
+%pre
+<%= pre %><% } %>
+
+<% if (preun) { %>
+%preun
+<%= preun %><% } %>
+
 %post
+<% if (post) {%><%= post %>
+<% } else { %>
 update-desktop-database &> /dev/null
+<% } %>
 
 %postun
+<% if (postun) {%><%= postun %>
+<% } else { %>
 update-desktop-database &> /dev/null
```

Comment:
Might want to specify in the docs that they'll need to update the desktop file cache themselves if they specify `post`/`postun` scripts.

Reply:
Updating the cache is only necessary if the software is handling some MIME types, which is not always the case and hence this should not always be called. I was actually considering removing the line altogether. On `-debian` we never had scripts until recently and hence we never called this and we still don't call this function even if the user doesn't specify scripts. I'm leaning towards removing the line. @malept thoughts?

Reply:
Perhaps add the line only if `options.mimeType` is passed in?

Reply:
That leaves us with a special cases when the user passes it's own script and also has MIME types. Do we append the command to the user script or just use the user script?
I think it's better to delete the line, that way this module works the same as `-debian`. I'm going to go with your original suggestion, of putting a note under `options.mimeType` that if it's used, `update-desktop-database &> /dev/null` should be added as `post` and `postun` scripts